tests/test_dataclasses.py

```diff
@@ -2269,16 +2269,30 @@ class TestPymatgenFromStructureData:
     """
 
     @skip_pymatgen
-    def test_1(self):
+    @pytest.mark.parametrize(
+        'pbc', ((True, True, True), (True, True, False), (True, False, False), (False, False, False))
+    )
+    def test_1(self, pbc):
         """Tests the check of periodic boundary conditions."""
-        struct = StructureData()
-        struct.set_cell([[1, 0, 0], [0, 1, 2], [3, 4, 5]])
-        struct.pbc = [True, True, True]
-        struct.get_pymatgen_structure()
+        import numpy as np
 
-        struct.pbc = [True, True, False]
-        with pytest.raises(ValueError):
-            struct.get_pymatgen_structure()
+        cell = np.diag((1, 1, 1)).tolist()
```

Comment:
Please add a test case for pbc=(False, False, False) which should allow `cell = None`.

Reply:
Thanks for the head up, I added now a new logic. If you had the time to review it, it would be great, thanks!